File: db/parseViews.php
<?php

$fi = fopen($argv[1],'r');
$fo = fopen($argv[2],'w');
while ($line = fgets($fi)) {
  echo($line);
  $sl = substr($line,0,9);
  if( 0 == strcmp($sl,'*********') ) {
    fwrite($fo, '-- ---'.substr($line,-1));
  } elseif( 0 == strcmp($sl,'  notice:') ) {
    fwrite($fo, substr($line,10));
  } elseif( 0 == strcmp($sl,' drop_vw:') ) {
    fwrite($fo, substr($line,10));
  } elseif( 0 == strcmp($sl,'view_ddl:') ) {
    fwrite($fo, substr($line,10));
  } else {
    fwrite($fo, $line);
  }
}
fclose($fi);
fclose($fo);
?>

File: db/ParseViews.py
'''
Created on Nov 22, 2018

@module: 
@author: Allan

Description: 
'''
import sys

args = sys.argv
#print( args )

fi = open(args[1], "r")
fo = open(args[2],'w')

for line in fi :
    print (line)
    sl = line[0:9]
    if sl == '*********' :
        fo.write( '-- ---' + line)
    elif sl == '  notice:' :
        fo.write( line[10:] )
    elif sl == ' drop_vw:' :
        fo.write( line[10:] )
    elif sl == 'view_ddl:' :
        fo.write( line[10:] )
    else :
        fo.write( line )


fi.close()
fo.close()